src/wp-admin/js/comment.js

```diff
@@ -32,6 +56,13 @@ jQuery(document).ready( function($) {
 		event.preventDefault();
 	});
 
+	/**
+	 * @summary Sets the time stamp values when the ok button is clicked.
```

Comment:
Probably not a problem, but inconsistent description with the listener above this one. 
first one: "when 'cancel' is clicked".
second one: "when the ok button is clicked".